iso_config.yaml

```diff
@@ -0,0 +1,37 @@
+log_file: "/var/tmp/host-os-iso-creator.log"
+mock_config: "./extras/centOS/7.2/mock/spin-iso-epel-7-ppc64le.cfg"
+rpm_dir: "./results"
```

Comment:
Initially, I've thought about it but as we decided to keep to rpm package builder and ISO creator code separate, I kept the same approach for the yaml file.

If the team agrees with this proposal, I don't see any problem in merger both yaml files as you suggest.